src/io/flutter/editor/FlutterColors.java

```diff
@@ -97,8 +97,8 @@ else if (colors.containsKey(key + primarySuffix)) {
   /**
    * Returns the the shortest material color name matching a color if one exists.
    */
-  public static @Nullable
-  String getColorName(Color color) {
+  @Nullable
+  public static String getColorName(@Nullable Color color) {
     String name = colorToName.get(color);
```

Comment:
will `colorToName.get(color)` throw an exception if `color` is null?

Reply:
I don't believe so - it'll just return `null`.